### DISCUSSION TITLE
Hello?

### DISCUSSION THREAD

**Discussion by joeyoropesa-dev** [reiryuki/Miui-Core-Magisk-Module#28] [Q&A]
Did anyone care about issues that was made using issues and after that moved here

And did anyone is checking discussions tab?

It seems that it takes months for someone to see and react but anyway it's still nothing

Are we've moved to discussions tab and this place is getting checked day-by-day or months-by-months or this project is discontinued and out-of-support for new issues that was made no matter that we've used issues tab or discussions tab

Thanks in advance for any response if any response happens sooner or later.

**Comment by reiryuki** [comment]
I'm sorry, my time is limited so I will only answer the important things.
